specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "App Compliance Automation Tool for Microsoft 365",
+    "version": "2022-11-16-preview",
+    "description": "App Compliance Automation Tool for Microsoft 365 API spec"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.AppComplianceAutomation/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "AppComplianceAutomation"
+        ],
+        "description": "Query the ongoing operation status.",
+        "operationId": "OperationResults_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded."
+          },
+          "202": {
+            "description": "Accepted. The request is still running."
+          },
+          "204": {
```

Comment:
I dont think you need the 204. Id a resource is not present then the delete should fail immediately. No need to start a long running operation.

Reply:
@raosuhas Yes, we can remove the Long-running operation for the GET operation.